acme/resource_acme_certificate_test.go

```diff
@@ -388,6 +407,17 @@ func testAccPreCheckCertZoneID(t *testing.T) {
 	}
 }
 
+func testAccPreCheckCertMultiProviders(t *testing.T) {
+	if v := os.Getenv("ACME_MULTI_PROVIDERS"); v == "" {
+		t.Skip("ACME_MULTI_PROVIDERS must be set for the multiple providers certificate acceptance test")
+	} else {
+		providers = strings.Split(os.Getenv("ACME_MULTI_PROVIDERS"), ",")
+		if len(providers) != 2 {
+			t.Fatal("ACME_MULTI_PROVIDERS must specify exactly two providers")
```

Comment:
`t.Skip` should be used here so that it doesn't give a `FAIL` result if the environment variable is omitted.

Reply:
I skip on line 412 if the environment variable is omitted. I only fail if the variable is supplied, but in a format incompatible with the test. I can change if this is not ideal.

Reply:
Wow! Yeah my bad. Carry on :+1: